fix pre-commit

AlxShvalev opened the pull request:
Fix pre-commit errors

AlxShvalev committed:
fix pre-commit

Fix pre-commit errors

NataliyaPavlova commented on setup.cfg:
```diff
@@ -1,15 +1,13 @@
-profile = "black"
+[flake8]
+exclude =
+    venv/,
```
1) файлы .env, .evn.example я бы тоже добавила в exclude
2) и еще __ pycache __, .git

AntonZelinsky replied:
Туда флак бы и не полез, но раз удалили, то лишним не будет)

NataliyaPavlova commented on setup.cfg:
```diff
@@ -1,15 +1,13 @@
+    venv/,
+    env/,
+    tests/,
+    migrations/
```
путь к миграциям некорректный - эта дира на другом уровне

NataliyaPavlova commented on setup.cfg:
```diff
@@ -1,15 +1,13 @@
+    tests/,
+    migrations/
+max-line-length = 120
+
```
еще у флейка есть полезный флаг --ignore, посмотри, может, нам тоже надо: https://github.com/Studio-Yandex-Practicum/Lubimovka_backend/blob/develop/setup.cfg

AlxShvalev committed:
fix/pre-commit

added ignore section; .env, .env.example, .git, __pycache__ in exclude section; max-complexity in flake8 config setup.cfg

AntonZelinsky commented on setup.cfg:
```diff
@@ -1,15 +1,28 @@
+exclude =
+    tests/,
+    migrations/
+profile = black
```
Блек же вроде бы удалили в этом ПРе? Или это не про это?
Почему, кстати, решили отказаться от чёрного?

AlxShvalev replied:
В данном случае black - это встроенный профиль isort. Сам black удалил т.к. не смог пока запустить его с конфигурационным файлом. Работает с pyproject.toml, но не хочет работать с setup.cfg. Вообще, как я понимаю, инструменты типа pre-config, isort, black, flake8 и т.д. нужны только при разработке и не нужны на проде. Наверное имеет смысл отдельные конфигурационные файлы иметь для прода и для дева.

AntonZelinsky reviewed (approved):
Сделайте пост о том, что нужно установить у себя прекомиты и попросите поставить плюсики кто сделал, чтобы у всех были одинаковые настройки
